CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# CONTRIBUTING
+---
+**To contribute**
```

Comment:
Moved here from the design doc. This file is a better place for contribution guidelines.

Reply:
I forgot this exists! lol
Ty!